gammapy/modeling/parameter.py

```diff
@@ -115,13 +117,17 @@ def __init__(
         scale_method="scale10",
         interp="lin",
         prior=None,
+        scale_transform="lin",
     ):
         if not isinstance(name, str):
             raise TypeError(f"Name must be string, got '{type(name)}' instead")
 
         self._name = name
         self._link_label_io = None
-        self.scale = scale
+        self._scale_method = scale_method
+        self._scale_transform = scale_transform
+        self.interp = interp
+        self._scale = float(scale)
```

Comment:
I must say that `Parameter` argument have become very complex. Is there a use case for a different scaling for scan and for parameter transform?
Is `scale_method` only relevant for `scale_transform=="lin"`?

Reply:
the scale can also be applied `scale_transform!="lin"` but I have no idea if this is useful.